service/README.md

```diff
@@ -36,6 +36,33 @@ For more technical details about how configuration is resolved you can read the
 
     `./otelcorecol --config=file:examples/local/otel-config.yaml --config="yaml:exporters::logging::loglevel: info"`
 
+
+# How to check components available in a distribution
+ 
+Use the sub command build-info. Below is an example:
+
+```bash
+   .\otelcol build-info
```

Comment:
All the other examples use `./otelcorecol` we either change that or this.

Reply:
I would change this thanks